bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/DmxHandlerFactory.java

```diff
@@ -36,15 +42,22 @@
  * @author Jan N. Klug - Initial contribution
  */
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.dmx")
+@NonNullByDefault
 public class DmxHandlerFactory extends BaseThingHandlerFactory {
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Stream
+            .of(ArtnetBridgeHandler.SUPPORTED_THING_TYPES, Lib485BridgeHandler.SUPPORTED_THING_TYPES,
+                    SacnBridgeHandler.SUPPORTED_THING_TYPES, ChaserThingHandler.SUPPORTED_THING_TYPES,
+                    ColorThingHandler.SUPPORTED_THING_TYPES, DimmerThingHandler.SUPPORTED_THING_TYPES,
+                    TunableWhiteThingHandler.SUPPORTED_THING_TYPES)
+            .flatMap(Set::stream).collect(Collectors.toUnmodifiableSet());
```

Comment:
Consider `Set.of()`.

Reply:
That's not possible, we'll end up with a `Set<Set<ThingTypeUID>>` then.

Reply:
Oh, you are right, was reading too fast.